com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfInertial.cs

```diff
@@ -46,7 +46,7 @@ public static void Create(GameObject linkObject, Link.Inertial inertial = null)
 #endif
             if (inertial != null)
             {
-                robotLink.mass = (float)inertial.mass;
+                robotLink.mass = ((float)inertial.mass > 0)?((float)inertial.mass):(float)0.1;
```

Comment:
Please parameterize the default value 0.1 as the private variable as `RoundDigits` and `MinInertia`